controllers/user.mjs

```diff
@@ -0,0 +1,7 @@
+import users from '../models/user';
+
```

Comment:
This is not a controller in any sense. Naming this as `userService` would be more adequate.

Reply:
Yip. I haven't given it much thought until now. I shall change it in a bit